app/helpers/policy_dashboard_helper.rb

```diff
@@ -8,7 +8,7 @@
 # along with this software; if not, see http://www.gnu.org/licenses/gpl.txt
 #
 
-module ScaptimonyPolicyDashboardHelper
+module PolicyDashboardHelper
   Colors = {
```

Comment:
I know it's completely unrelated but could I kindly ask you to fix also this one? It uses the same colors so you could either define the constant like this
```ruby
COLORS = {
  :compliant_hosts => ArfReportDashboardHelper::COLORS[:passed],
  :incompliant_hosts => ArfReportDashboardHelper::COLORS[:failed],
}
```

or use ArfReportDashboardHelper::COLORS in this helper directly.